lib/mhv_logging/configuration.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+require 'rx/configuration'
+module MHVLogging
+  # Configuration class used to setup the environment used by client
+  class Configuration < Rx::Configuration
```

Comment:
How can it be a subclass of Rx::Configuration when things like the base path are pretty different between the two?